docs/30-sonata-media-private-file-provider.md

```diff
@@ -0,0 +1,31 @@
+Sonata Media Private File Provider
+==================================
+
+This file provider extends the file provider of [Sonata Media package][1] and allow to store
+media files on a non public place.
+
+The main purpose of this provider is to store media that are private and should not have 
+a public url.
+
+Configuration
+-------------
+
+```yaml
+sonata_helpers:
+    sonata_media_private_file_provider:
+        url_prefix: /admin/private #default
+        storage_path: data/media #default - will be concatenated with kernel.project_dir
```

Comment:
I think you should add `%kernel.project_dir%/` to be able to configure something like `/data/foo/bar`